hw#6.1.py

```diff
@@ -0,0 +1,32 @@
+import datetime
+
+
+def moto_time(inp_m):
+    if not 0 < inp_m < 1440:
+        raise ValueError
+    inp_h = 0
+    while inp_m >= 60:
+        inp_m -= 60
+        inp_h += 1
+    ans_m = list(map(int, list(str(inp_m))))
+    ans_h = list(map(int, list(str(inp_h))))
+    print(datetime.time(hour=inp_h, minute=inp_m), sum(ans_h + ans_m))
+
+
+def level_up(experience: int, threshold: int, reward: int) -> None:
+    print(experience + reward == threshold)
+
+
+def time_converter(time: str):
+    hour: int = int(time.split(':')[0])
+    minute = time.split(':')[1]
```

Comment:
If you selected an approach, pls be consistent 
```suggestion
    minute: str = time.split(':')[1]
```